Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.12.5-alpine
+
+RUN apk --update add openjdk8-jre-base git
+
+WORKDIR /usr/scr/app
```

Comment:
I can't believe I missed this, but it should be `src` (as in source) not `scr` (as in screen)